community-build/src/scala/dotty/communitybuild/projects.scala

```diff
@@ -127,19 +144,19 @@ object projects:
   lazy val ujson = MillCommunityProject(
     project = "upickle",
     baseCommand = s"ujson.jvm[$compilerVersion]",
-    dependencies = List(scalatest, scalacheck, scalatestplusScalacheck, geny)
```

Comment:
Why are scalatest/scalacheck deps removed?

Reply:
This is mentioned in the commti message:
> - upickle/ujson do not depend on scalatest when building with
>  dotty currently so we can drop the dependency in projects.scala.